components/monitoring/prometheus/base/kustomization.yaml

```diff
@@ -4,9 +4,9 @@ resources:
 - install-prometheus.yaml
 - prometheus-view.yaml
 - configure-prometheus.yaml
-- prometheus-servicemonitors.yaml
+- servicemonitors/
```

Comment:
you need to include each file or have servicemonitors/kustomization.yaml (where still all files must be listed). Now kustomization is failing:
```
$ kustomize build components/monitoring/prometheus/base/
Error: accumulating resources: accumulation err='accumulating resources from 'servicemonitors/': '/home/mkovarik/git/appstudio/infra-deployments/components/monitoring/prometheus/base/servicemonitors' must resolve to a file': couldn't make target for path '/home/mkovarik/git/appstudio/infra-deployments/components/monitoring/prometheus/base/servicemonitors': unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/home/mkovarik/git/appstudio/infra-deployments/components/monitoring/prometheus/base/servicemonitors'
```